src/ir/context.rs

```diff
@@ -2160,16 +2160,21 @@ If you encounter an error missing from this list, please file an issue or a PR!"
             ::clang_sys::CXCursor_Namespace,
             "Be a nice person"
         );
+
+        let mut module_name = None;
+        let spelling = cursor.spelling();
+        if !spelling.is_empty()
+        {
```

Comment:
(I don't think this brace really needs to live on a different line, but it's alright in any case)